hardware_interface/source/eps.c

```diff
@@ -87,6 +87,30 @@ void eps_set_pwr_chnl(uint8_t pwr_chnl_port, bool bit){
 
 }
 
+/**
+ * @brief Convert 64-bit number from host byte order to little endian byte order
+ * @attention csp_letoh64 does not work correctly. Moving this function to
+ * csp_endian also returns a wrong value. The reason is probably the limitations
+ * of MCU on processing double precision floats.
+ */

```

Comment:
@arrooney Any thoughts? (Also check the conversation tab. It's not showing here anymore since it's outdated.)

Reply:
I don't have thoughts, as long as it's been tested with all sorts of values?